raven/processes/wps_raven.py

```diff
@@ -93,6 +93,7 @@ def options(self, request):
                 else:
                     data = obj.data
 
+                # FIXME: Breaking changes in RavenPy occurred at https://github.com/CSHS-CWRA/RavenPy/commit/8e21efb8e2f0e16a3e1a5fedeec098a4fc3c5d98
```

Comment:
@huard

I'm not sure how best to proceed here. This change is causing the majority of errors for the time being. I won't pretend to understand how to proceed on this.